src/CMakeLists.txt

```diff
@@ -732,12 +768,19 @@ if(${WITH_RBD})
   set(DENCODER_EXTRALIBS
       ${DENCODER_EXTRALIBS}
       rbd_replay_types)
+  set(dencoder_srcs
+      ${dencoder_srcs}
+      $<TARGET_OBJECTS:krbd_objs> )
 endif(${WITH_RBD})
+if(${WITH_KVS})
+  set(dencoder_srcs
```

Comment:
On 4-7-2016 06:14, Kefu Chai wrote:
> In src/CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10024#discussion_r69407011>:
> 
>>  endif(${WITH_RBD})
>> +if(${WITH_KVS})
>> +  set(dencoder_srcs
> 
> might want to use |list(APPEND| instead.

That are thing I still have to figure out.
Sometimes |list(APPEND| is usable, but only when cmake uses the list.
Otherwise it is a list wit ';' between the elements, and that does not
work for the other tools.

Same is a bit when to use if(${VAR}) and when if(VAR), but the first
case always seem to work. Which is why I'm using that one.
Got bitten once to many having to rerun
Guess I really do need to read up in that.